### PR TITLE
nodejs-select2 3.5.2

### PR DESCRIPTION
**Pull request by dLobatog**
Depends on https://github.com/theforeman/foreman/pull/3433

### CODE REVIEW

File: nodejs-select2/nodejs-select2.spec [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+%install
+mkdir -p %{buildroot}%{nodejs_sitelib}/%{npm_name}
+cp -pfr *.png *.css *.json *.md *.js %{buildroot}%{nodejs_sitelib}/%{npm_name}
+
+%nodejs_symlink_deps
+
+%check
+
+%clean
```
Comment (domcleal) [inline nodejs-select2/nodejs-select2.spec]: Not required.

File: nodejs-select2/nodejs-select2.spec [hunk 2]
```diff
@@ -0,0 +1,67 @@
+# TODOs:
```
Comment (domcleal) [inline nodejs-select2/nodejs-select2.spec]: Remove these?

File: nodejs-select2/nodejs-select2.spec [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+#   - list doc files properly (%doc %{nodejs_sitelib}/%{npm_name}/README.md)
+#   - exclude unnecessary files from %files section
+
+%global npm_name select2
+
+Name: nodejs-%{npm_name}
+Version: 3.5.2
+Release: 1%{?dist}
+Summary: Browserify-ed version of Select2.
+License: unknown
```
Comment (domcleal) [inline nodejs-select2/nodejs-select2.spec]: 3.5.x appears to be dual-licensed as Apache 2 or GPLv2 (https://github.com/select2/select2/blob/stable/3.5/LICENSE), so `ASL 2.0 or GPLv2`

**Comment by dLobatog** [comment]
Rebased

**Comment by domcleal** [comment]
Merged as ed359d1, thanks @dLobatog.

**Pull request closed without merging**